2/contrib/openshift/base-plugins.txt

```diff
@@ -28,17 +28,18 @@ openshift-login:1.0.27
 openshift-sync:1.0.53
 pam-auth:1.6
 parameterized-trigger:2.43.1
-pipeline-build-step:2.16
-pipeline-utility-steps:2.12.0
+pipeline-build-step:2.18
+pipeline-input-step:449.v77f0e8b_845c4
+pipeline-utility-steps:2.13.0
 prometheus:2.0.10
 scm-api:2.6.5
-script-security:1131.v8b_b_5eda_c328e
-snakeyaml-api:1.29.1
+script-security:1158.v7c1b_73a_69a_08
+snakeyaml-api:1.30.1
```

Comment:
Are these updated due to transitive dependencies?

Reply:
Yes, I will update in a separate commit